### PR TITLE
DSTU2 Infrastructure Inconsistencies update

### PR DESCRIPTION
**Pull request by CBrashear**
Went through Infrastructure for DSTU2 to look for inconsistencies.

Binary:
- Added disclaimers

Document Reference:
- Added disclaimers
- Moved create around for consistency placement with other resources
DocRef_TOC_update
- Changed to snipped for brevity in some examples
DocRef_brevity_update

### CODE REVIEW

File: content/millennium/dstu2/infrastructure/document-reference.md [hunk 1]
```diff
@@ -92,8 +92,43 @@ Notes:
...
 <%= headers status: 200 %>
-
 <%= json(:dstu2_document_reference_docref_search) %>
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
```
Comment (FFSol) [inline content/millennium/dstu2/infrastructure/document-reference.md:102]: Good catch

File: content/millennium/dstu2/infrastructure/document-reference.md [hunk 2]
```diff
@@ -92,8 +92,43 @@ Notes:
 #### Response
 
 <%= headers status: 200 %>
-
```
Comment (wseymour15) [inline content/millennium/dstu2/infrastructure/document-reference.md:95]: nit: any reason you removed this?

File: content/millennium/dstu2/infrastructure/document-reference.md [hunk 3]
```diff
@@ -159,6 +193,7 @@ _Implementation Notes_
    x-runtime → 5.497541
    x-xss-protection → 1; mode=block
 </pre>
```
Comment (wseymour15) [inline content/millennium/dstu2/infrastructure/document-reference.md:195]: nit: maybe add a newline here

File: content/millennium/dstu2/infrastructure/document-reference.md (hunk 2 above)
Comment (CBrashear) [reply content/millennium/dstu2/infrastructure/document-reference.md:95]: All the other response calls in this file have been squished together, just being consistent.

Comment (wseymour15) [reply content/millennium/dstu2/infrastructure/document-reference.md:95]: fair enough, +1

**Comment by MaxPhilips** [comment]
Screen Shot 2020-04-07 at 5 26 07 PM
Screen Shot 2020-04-07 at 5 26 22 PM
validated deployment

**Pull request merged**